examples/advanced/nce_example.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/python3
```

Comment:
Fantastic!!!

chose another name though, NCE = Noise Contrastive Estimation for us...

counterfactual_explain or explain_obj_counterfactual? Something...